docs/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # What's New with Enterprise-NG
 
+## v7.0.0
```

Comment:
Ok sorry if i gave mixed message but we decided master can be 6.3.x for now. I think this new section might be a problem as we merge earlier PR's and need adjusting.